packages/document/builder-doc/docs/zh/shared/rspackTip.mdx

```diff
@@ -1,4 +1,4 @@
-:::tip 什么是 Rspack
+:::tip Rspack - 高性能 Web 构建工具
```

Comment:
we should also modify `packages/document/builder-doc/docs/en/shared/rspackTip.mdx` & `packages/document/builder-doc/docs/en/shared/rspackPrecautions.md` about this part